build.gradle

```diff
@@ -48,6 +48,7 @@ ext {
           zipkinBrave          : 'io.zipkin.brave:brave:4.17.2',
           retrofit             : 'com.squareup.retrofit2:retrofit:2.3.0',
           retrofitMoshi        : 'com.squareup.retrofit2:converter-moshi:2.3.0',
+          retrofitWire         : 'com.squareup.retrofit2:converter-wire:2.4.0',
```

Comment:
should this be 2.3.0 as the other retrofit deps?  or conversely, bump the others?